### PR TITLE
nuked

### PR DESCRIPTION
**Pull request by tadad**
**Problem**:
separating dev-suite and uqbar-core

**Solution**:
delete everything

**Notes**:
cleaning up some other stuff while I see it

### CODE REVIEW

**Comment by tadad** [comment]
@hosted-fornet let me know if you want to delete ziggurat here or on another branch

**Comment by hosted-fornet** [comment]
> @hosted-fornet let me know if you want to delete ziggurat here or on another branch

@tadad Sorry I didn't see this till now: yes, please do delete %ziggurat here.

File: mar/indexer/headers.hoon [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /-  ui=zig-indexer,
-    zig=zig-ziggurat
+    zig=zig-ziggurat :: TODO
```
Comment (tadad) [inline mar/indexer/headers.hoon:2]: @hosted-fornet - not sure what to do here? an indexer mark shouldn't really have a ziggurat dependency, but I'll let you sort this out

**Comment by tadad** [comment]
also now that I think about it, our desk.docket-0 might need to change as well? And the frontend obviously though maybe @0x70b1a5 is already on it?

**Pull request merged**